wsgi/odyssy/basic/templates/basic/index.html

```diff
@@ -71,7 +74,12 @@
                                     </span>
                                 </div>
                                 <!-- Event-Description -->
-                                <div class="event-text"><a href="{% url 'event-view-single' event.id %}">{{ event.name }}</a></div>
+                                <div class="widgets-item-text">
+                                    <a class="widgets-item-title" href="{% url 'event-view-single' event.id %}">{{ event.name }}</a>
+                                    <div class="widgets-item-description">
+                                        {{ event.description|safe|truncatewords_html:"10"|linebreaks }}
```

Comment:
In Django documentation there's no space arround `|`. Take a look, search for 
`{% firstof var1 var2|safe var3 "<strong>fallback value</strong>"|safe %}`
https://docs.djangoproject.com/en/1.11/ref/templates/builtins/